src/fairseq2/assets/cards/datasets/librilight.yaml

```diff
@@ -6,3 +6,4 @@
 
 name: librilight_asr_10h
 dataset_family: generic_asr
+data: /checkpoint/mms/shared/assets/debug/librilight
```

Comment:
to rm

Reply:
The open-source recipes should run everywhere with internet, and additional paths for RSC are provided through fairseq2-ext (as they are not specific to mms itself).

Custom mms assets (this is not one of them, but just for reference in the future), should live in the mms repository. 

Therefore, as all assets here are for the open-source recipes, they should be included in fairseq2-ext, which I've already prepared a branch for https://github.com/fairinternal/fairseq2-ext/blob/cirquit/v0.5/src/fairseq2_ext/cards/models/wav2vec2.yaml .

Will update this after we migrate to v0.5 officially.